sbi/utils/user_input_checks.py

```diff
@@ -530,21 +530,25 @@ def check_estimator_arg(estimator: Union[str, Callable]) -> None:
     )
 
 
-def validate_theta_and_x(theta: Any, x: Any) -> None:
+def validate_theta_and_x(theta: Any, x: Any) -> bool:
```

Comment:
I suggest removing the return and thus also the type?

Reply:
no problem

Reply:
done with 96c2b0c